src/Gravatar.php

```diff
@@ -29,6 +71,10 @@
      */
     private $secure;
 
+    /**
+     * @param array $defaults
```

Comment:
Docblocks are intentionally removed, please do not add them back.

Reply:
I have updated the PR to remove the doc blocks.